cupy/cutensor.pyx

```diff
@@ -107,7 +133,8 @@ def _set_compute_dtype(array_dtype, compute_dtype=None):
     return compute_dtype
 
 
-def create_tensor_descriptor(a, uop=cutensor.OP_IDENTITY):
+cpdef TensorDescriptor create_tensor_descriptor(
+        ndarray a, int uop=cutensor.OP_IDENTITY, Handle handle=None):
```

Comment:
How about making `handle` as a keyword argument and name it `_handle` to make it clear it is a private interface?

Reply:
We can't use a keyword-only argument for `cpdef` or `cdef` function.